### ISSUE TITLE
Horizontal scroll issue!

### ISSUE THREAD

**Issue by waqar-imtiaz** [michalsnik/aos#227]
So this happens on small screens
![image](https://user-images.githubusercontent.com/3416177/30899972-a07b8948-a37b-11e7-9443-017cb997d89c.png)

and it auto resolves when we scroll to that section and in the end it's fixed, like this
![image](https://user-images.githubusercontent.com/3416177/30900020-d15f3dfc-a37b-11e7-82d8-2263d3d960f3.png)

how can I fix it? I understand that its causing because of transforms that are used for animation purpose but is anyway around it that it does cause overflow? it's effecting the fixed position elements. 

P.S: Sorry if this is not the right way of asking, I never posted like ever :)

**Comment by jonathanj** [comment]
This issue was closed but unfortunately there was no reason for closing it, I'm having the same issue.

**Comment by waqar-imtiaz** [comment]
I have reopened the issues, it's causing because of transforms on x axis, if you don't use any such properties like slideLeft or flipLeft, fadeLeft. and the scroll goes away as you reached to that section.

**Comment by jonathanj** [comment]
@waqar-imtiaz Indeed! I did some digging through other issues and applying `overflow-x: hidden` to the relevant container element (which ended up not being the top-level container, weirdly enough) resolved the issue for me.

One of the comments mentioned adding this to the FAQ, which would probably help.

**Comment by Dzordzu** [comment]
Solution in #234

**Comment by nabtron** [comment]
Adding `overflow-x: hidden` to the element container broke the (smooth) scroll manually, so I placed `overflow-x: hidden` to the body and it works perfectly now!

**Comment by 1SudhanshuSingh** [comment]
html, body {
    overflow-x: hidden;
}

**Comment by ichsanputr** [comment]
sure this solution works 100%

add this code n your file css

`html, body {
overflow-x: hidden;
}`

**Comment by tutods** [comment]
But if I remove the overflow-x on html the fade-right and fade-left doesn't work for me 😟

**Comment by nabtron** [comment]
@tutods did u try on body?

**Comment by tutods** [comment]
Yes but the horizontal scroll remains (most on mobile)

**Comment by mr-alipour** [comment]
html, body { overflow-x: hidden; }

**Comment by mr-alipour** [comment]
This css code solve this problem perefectly
html, body { overflow-x: hidden; }
